NuGet.config

```diff
@@ -12,6 +12,7 @@
     <add key="darc-pub-dotnet-emsdk-3df5eac" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-3df5eac9/nuget/v3/index.json" />
     <!--  End: Package sources from dotnet-emsdk -->
     <!--  Begin: Package sources from dotnet-sdk -->
+    <add key="darc-pub-dotnet-sdk-fe6d1ce" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-sdk-fe6d1ced/nuget/v3/index.json" />
```

Comment:
We're taking a dep on an unreleased version of MicrosoftDotNetApiCompatTaskVersion here which is what is causing WasmBuildTests to fail.  If we want to do this then WBT needs to have the stable feed added to its build as well

Reply:
Is this commit I pushed enough to fix the failure? https://github.com/dotnet/runtime/pull/114662/commits/ab101575876d3e1691bd967392b064f5eb8fb882

Reply:
Okay I see some linux wasm legs passing already. Merging now.

Reply:
I think you didn't wait long enough.